aries_cloudagent/protocols/present_proof/v1_0/routes.py

```diff
@@ -427,61 +427,6 @@ class V10PresentationRequestSchema(Schema):
     return web.json_response(presentation_exchange_record.serialize())
 
 
```

Comment:
I'm not sure if `presentation_exchange_create_request` was removed intentionally, but I believe we use that in identity kit to create presentation exchanges that can be responded to without a connection.

Reply:
My mistake, I had thought it redundant vs. ...send...(). I am fixing and amending the PR.